src/hangar/repository.py

```diff
@@ -651,6 +651,12 @@ def create_branch(self, branch_name, base_commit=None):
         bool
             if the operation was successful.
         '''
+        if not is_ascii_alnum(branch_name):
```

Comment:
I could move this to a separate PR if you want to keep this PR just for test cases